src/copas.lua

```diff
@@ -597,15 +597,18 @@ function copas.addserver(server, handler, timeout)
     end
 end
 
-function copas.removeserver(server)
+function copas.removeserver(server, close_it)
```

Comment:
I wanted to avoid a negated name, which is sometimes confusing especially when it causes double negation such as `no_close=false`. 

But an easy way around this is to call it `keep_open` :)